tests/programs/programs.js

```diff
@@ -63,6 +65,85 @@ module.exports = {
       .end();
   },
 
+  'Join a Program with wrong email': browser => {
+    let inviteId;
+
+    loginAsUser(browser)(TEST_USERS.DCC_ADMIN)
+      .url(buildUrl(`/submission/program/${program.shortName}/manage?tab=users`))
+      .click('#add-users')
+      .setValue('[aria-label="First name"]', 'admin')
+      .setValue('[aria-label="Last name"]', 'single')
+      .setValue('[aria-label="Email"]', TEST_USERS.PROGRAM_ADMIN_SINGLE.email)
+      .click('#modal-add-users');
+
+    browser
+      .url(process.env.MAILHOG_ROOT)
+      .click('xpath', `//div[contains(text(), '${TEST_USERS.PROGRAM_ADMIN_SINGLE.email}')][1]`)
+      .frame('preview-html', function() {
+        this.getAttribute('xpath', "//a[contains(text(), 'JOIN THE PROGRAM')]", 'href', r => {
+          inviteId = r.value.match(/[^\/]*$/)[0];
+          this.url(buildUrl(`/submission/program/join/login/${inviteId}`))
+            .waitForElementVisible('#google-login', 10000)
+            .click('#google-login')
+            .deleteCookies()
+            .refresh()
+            .waitForElementVisible('input[type="email"]')
+            .setValue('input[type="email"]', TEST_USERS.PROGRAM_ADMIN_MULTI.email)
+            .click('#identifierNext')
+            .waitForElementVisible('input[type="password"]')
+            .setValue('input[type="password"]', TEST_USERS.PROGRAM_ADMIN_MULTI.pass)
+            .click('#passwordNext')
+            .useXpath()
+            .waitForElementVisible(`//*[contains(text(), 'Incorrect email address')]`);
+        });
+      })
+      .end();
+  },
+
+  'Join a Program': browser => {
+    let inviteId;
+
+    loginAsUser(browser)(TEST_USERS.DCC_ADMIN)
+      .url(buildUrl(`/submission/program/${program.shortName}/manage?tab=users`))
+      .click('#add-users')
+      .setValue('[aria-label="First name"]', 'admin')
+      .setValue('[aria-label="Last name"]', 'single')
+      .setValue('[aria-label="Email"]', TEST_USERS.PROGRAM_ADMIN_SINGLE.email)
+      .click('#modal-add-users');
+
+    browser
+      .url(process.env.MAILHOG_ROOT)
+      .click('xpath', `//div[contains(text(), '${TEST_USERS.PROGRAM_ADMIN_SINGLE.email}')][1]`)
+      .frame('preview-html', function() {
+        this.getAttribute('xpath', "//a[contains(text(), 'JOIN THE PROGRAM')]", 'href', r => {
+          inviteId = r.value.match(/[^\/]*$/)[0];
+          this.url(buildUrl(`/submission/program/join/login/${inviteId}`))
+            .waitForElementVisible('#google-login', 10000)
+            .click('#google-login')
+            .deleteCookies()
+            .refresh()
+            .waitForElementVisible('input[type="email"]')
```

Comment:
are we navigating the google login UI here? I haven't tried that out before myself but just curious cause I thought that didn't work

Reply:
Yes, it's navigating to google login UI. The input does not appear when the test browser has Google cookie, but after deleting cookies and refreshing, the input would appear.

Reply:
gotcha! @joneubank would that work with our login test as well? could give it a shot if we haven't tried :)

Reply:
If this hack works then that is huge... I'll run a trial of this script on browserstack to see if it gets through the issue